src/cider/exec/nextgen/operators/AggregationNode.cpp

```diff
@@ -133,14 +149,38 @@ void AggTranslator::codegen(context::CodegenContext& context) {
   int32_t current_expr_idx = 0;
   for (const auto& expr : exprs) {
     auto agg_expr = dynamic_cast<const Analyzer::AggExpr*>(expr.get());
-    utils::FixSizeJITExprValue values(agg_expr->get_arg()->get_expr_value());
-
     auto cast_buffer = buffer->castPointerSubType(jitlib::JITTypeTag::INT8);
     auto val_addr_initial = cast_buffer + exprs_info[current_expr_idx].start_offset_;
     auto val_addr = val_addr_initial->castPointerSubType(
         exprs_info[current_expr_idx].jit_value_type_);
 
-    if (exprs_info[current_expr_idx].sql_type_info_.get_notnull()) {
+    // count special case, cause count not null dont have arguments
```

Comment:
Not exactly, ```count()``` and ```count(col)``` are different, the former will count all input rows but the latter only counts not-null rows.